nautobot/extras/views.py

```diff
@@ -1253,6 +1253,7 @@ class JobListView(generic.ObjectListView):
     filterset = filters.JobFilterSet
     filterset_form = forms.JobFilterForm
     action_buttons = ()
+    non_filter_params = "view"
```

Comment:
I wanted this value here to be a tuple but ruff refused :man_shrugging:

Reply:
Were you trying `("view")`? Try `("view",)` instead.

Reply:
That did the trick, thanks!